src/GrpcProtos/capture.proto

```diff
@@ -96,7 +96,7 @@ message InternedCallstack {
   Callstack intern = 2;
 }
 
-message InternedCallstackSample {
```

Comment:
Because the intern is the proto `InternedCallstack` right above, this is just referring to the intern, it's super confusing to name both the intern and the proto referring to the intern with the same "Interned" prefix.